runtime/android/core_internal/src/org/xwalk/core/internal/XWalkPreferencesInternal.java

```diff
@@ -132,6 +132,12 @@ String getStringValue() {
             "support-multiple-windows";
 
     /**
+     * The key string to enable/disable spatial navigation like a TV controller.
+     */
+    @XWalkAPI
+    public static final String SPATIAL_NAVIGATION = "spatial_Navigation";
```

Comment:
You should follow the style of the other options above and call it `spatial-navigation` (or `enable-spatial-navigation` to be more clear) instead of `spatial_Navigation`.